sqrt.py

```diff
@@ -20,3 +20,6 @@ def main(number):
 
 if __name__ == "__main__":
   main(opt["--n"])
+
```

Comment:
wrong function, here is fix:

```suggestion
if __name__ == "__main__"
```